pkg/reconciler/serverlessservice/serverlessservice_test.go

```diff
@@ -769,9 +769,9 @@ func withHTTP2Priv(svc *corev1.Service) {
 	svc.Spec.Ports[0].Name = "http2"
 	svc.Spec.Ports[0].TargetPort = intstr.FromInt(networking.BackendHTTP2Port)
 
-	svc.Spec.Ports[4].Name = "http2-istio"
-	svc.Spec.Ports[4].Port = networking.BackendHTTP2Port
-	svc.Spec.Ports[4].TargetPort = intstr.FromInt(networking.BackendHTTP2Port)
+	svc.Spec.Ports[5].Name = "http2-istio"
```

Comment:
Could you add checks for the rest of the ports like: `http-queueadm`?

Reply:
I think this part overrides the port name. And the `func svcpriv` creates the ports like queueadm by default.